libensemble/logger.py

```diff
@@ -15,6 +15,19 @@ def manager_warning(self, message: str, *args, **kwargs) -> None:
 
 
 logging.Logger.manager_warning = manager_warning
+
+VDEBUG = 5
+logging.addLevelName(VDEBUG, "VDEBUG")
+logging.VDEBUG = VDEBUG
+
+
+def vdebug(self, message, *args, **kwargs):
+    if self.isEnabledFor(VDEBUG):
+        self._log(VDEBUG, message, args, **kwargs)
```

Comment:
CodeCov says that this line isn't covered, but I would think that the `test_logger.py` unit test would cover it.

Reply:
Test added.

Reply:
Test really added.